### DISCUSSION TITLE
Team Formation

### DISCUSSION THREAD

**Discussion by Heiwa1580** [Glimesh/company#38] [General]
I wanted to start this discussion around specifically the Discord channels in the Building-Glimesh section. We made the changes to the contributor section to try and help encourage more open collaboration and less private talks. 

I think we have some areas that we can improve on though. These are my opinions and not official statements. I wanted to open up the discussion to help us brainstorm on how we can improve the process altogether

1. I think that we need to establish teams a bit more specifically and that we should have a specific lead for these channels. Collaboration and group consensus are great and if a team decides that is how they will operate than there needs to be a specific standard on how that works. How that input is collected, but in that essence that process becomes the lead of the team. I think there has to be some kind of decision making process though that is determined or an individual who is placed in that position. If group consensus is chosen for a particular group I believe there should be one person designated as the spokesperson to help communicate and solidify the voice of the group.

2. I think we need to determine the proper use of the Building-Glimesh channels. Teams need to be able to define how the channel should work and what is appropriate for those channels and when to reach out to other teams channels. The lead of the team should be able to discuss that and those on the team would be the ones responsible for being organized about it.

3. I think that we need to determine one tool for all teams to work on collectively. At the moment it feels that we have too many tools and none of them do the job 100% and the current tools in use are all over and there is no clear standard on how we use them. I feel that the leads designated should work together and set a standard tool and standard method of working.

4. I think that we need to have more structured communications across teams. Our team meetings I think should be more open and collaborative and should be more about the team than the company. I think that we should have the team representatives bring what they are doing and report in on the progress of what they are working on. Then once a month we can go over the grand scheme of how Glimesh is doing (I believe SmileCode suggested this)

Overall I think we should have the discussion of how are we really going to work as a team and what we can do to improve our team work and define who we are and what we do as a team

**Comment by ghost** [comment]


**Reply by Missy305** [reply]
TLDR; I think today's discussion was mostly frustration from unaligned goals though was good for dynamic understanding. Also, the people I was in a discord channel with, who don't read the discord often, did during this and were impressed with GCT professionalism and defending company choices and values. The Glimesh team does a great job being transparent and opportunities for the community to be involved, maybe a more focused and strategized priority list to prevent future misunderstanding, and a lead per Building Glimesh section is a good idea. 

Following the lead idea, a pinned priority list or frequently pinged update of something like the Eisenhower Matrix (below) or what was done for Phase 2 would be beneficial in both efforts and reigning in rabbit trails. For example, GCX this weekend, the whole past week's conversation should have been nothing but GCX content and wanted impact, but conversation was still all over the place with individual's opinions and ideas on other topics despite efforts of GCT to keep it focused - including today taking away time from the start of GCX.
Trying to hit and utilize everyone's ideas is great, but there should be a roadmap for each "building Glimesh" section as well strategizing desires, ability, time, and overall impact from users to Glimesh and to users from the platform. I think the twitter thread is a great example of good strategy to have all of the basic "how to" made and pinned for a time; the impact of it coming for potential users because of GCX to lessen questions being better informed. 
Which brings me to I think a vote being called for in the building Glimesh channels for decisions and "what is best" after a disagreement or egos are hurt is an awful idea and easily influenced by talkative people. Previously made polls for feedback on Twitter, a discord bot, google form, etc - awesome. It should not be what makes the decision on a mild to major move. This needs to be from someone, or some people, who can see all the information needed and have a strategy in mind who can relay to a lead if the lead is not part of the "some people". A better use of voting is impact priority (like was done for Phase 2) or how an idea would be perceived (like GlimMovies's glimdrop choice).

![image](https://user-images.githubusercontent.com/86086614/122489345-14c5e880-cfa5-11eb-8f1b-773b27d1cba4.png)

The description for marketing involvement is:, "You can give ideas, share knowledge and or create and design items for us to use within our marketing campaigns." which is important to remember as a volunteer - it doesn't say ideas will be used or fully conversed right as a thought appears. Marketing needs a resource pool of ideas and items to pull as needed and the ideas should be flexible to be used, as needed. 


I'm not sure on #4 the team meeting being section focused unless it's done how the plotting of Phase 2 was done with the GCT and Contributors talking through processes. Making it open and involving chat would create another "too many cooks" and the Glimesh team talking about it would direct those with an idea to the channel already. Would it truly be an informational on how the company as a whole is doing then? Unless the company metrics and upcoming information summary is done at the start of each month on the previous month?

I think the Glimesh GCT and Contributors functions well as a team - it's expecting a public community to stay aligned to values and standards of the company rather than self being where the teamwork falls less than expectations. You will always have the creative extroverts who can't contain nor focus themselves who then crash into emotions, or you'll have trolls, and anywhere in between. However, books that have helped the culture where I work is Emotional Intelligence 2.0, The Ideal Team Player, and 5 Dysfunctions of a Team I would suggest if Glimesh teamwork is a concern. Making a Jahari window for Glimesh team and company may be beneficial for self discovery as well.


#3 What do you mean 1 tool? If software for designs or content, I think that would limit those working on the design if an unfamiliar tool is selected. Maybe an agreed quality standard instead? I suck at digital art though, so it may be fine one software to another for a professional or hobbyist.

**Reply by Heiwa1580** [reply]
So to clarify on the team meeting part

- The basic idea is that the team meeting would be driven by the leads or representatives of each section of the building Glimesh side of things. They would represent the voice of those involved and would share what the team is doing. The people involved wouldn't have to be same person each week mostly the point would be a weekly status / goals idea. 

- It wouldn't be so much about bringing the whole community into the meetings but more like getting more people involved who are doing the work. 

I imagine the meetings basically working with an agenda like this

- Each week set goals and check on last weeks goals
- Set a focus for the week
- Progress report on current projects
- Wins, challenges, open thoughts

The idea though would be this would be driven by those who are representing that sector. The sectors being the channels of the Building Glimesh sections that are applicable to the work. I think too we can organize the meeting in a way that we don't have to always go over every segment or have the same people. As long as we have some general checking in on the work and progress of Goals

We could probably come up with a better format but the idea would be making the meetings more about the work being done on Glimesh and the goals of Glimesh rather than a weekly growth check

**Reply by Missy305** [reply]
This makes sense.

**Comment by justsimy** [comment]
In the past few weeks, a lot of things for me have been very unclear. First, it was said we had to work in Marketing. Then it was Design. Then it was Business. Then it was Design and now it's in art and community design to a point that I'm lost. 100% lost. We need more clarification on this. We need one person we can count on that will review and approve each idea and until those ideas are voted on or 100% approved then we work on it. This is to ensure that we know. If someone works on something we give that person a chance to work on it and also give them the chance to ask for feedback. 

3. I'm not sure what you mean by this Heiwa. An issue with this idea is that if you tell designers to work in 1 specific program then you may get people who either can't afford something or do not know how to use the suggested software which may take away those designers that you need or want. This can send them away instead of keeping them.

2. An example for the design channel with this is for those who start working on design work whether graphics or sketches or whatnot for those to be shared as well as colors/layout and such so if someone is designing for example a Twitter post then we know what it will look like beforehand. Any designer with a professional background will know how to layout sketch. But they will also know the basics of whitespace/color theory and knowledge about color blindness and the standards for this.

**Reply by Heiwa1580** [reply]
The 1 tool thing I clarified below but it wasn't so much a designer to use one tool. More for all of us to have the same process of organizing work. That way all the teams are using the same methods and outlines as each other.

**Comment by Heiwa1580** [comment]
So to Clarify / rewrite point 3 (as rereading my thought I was thinking something different than what I wrote)

What I meant by one tool is currently we use Trello, Discord, Github, and Google Docs. On the Trello as well there are multiples of boards for several different team boards. But there is no one consistent way that these tools are used. Sometimes ideas are planned in Google Docs, sometimes they are written down in Discord and pinned, sometimes they end up in a Github discussion, or they are put in cards on Trello. 

I think what becomes difficult is that seeing the work that needs to be done or that is being done is difficult at times unless you are completely entrenched in the current Discord discussions daily. Also I think a lot of good ideas or thoughts are lost to Discord.

My suggestion is basically this. There should be one Glimesh process to how work is tracked and organized that all the teams should do their best to adhere to. Basically I think that anyone who knows the process should be able to see where work is happening, how it is happening and where it is in the process.

Ideas can be brought up in Discord. Fleshed out in Googles Docs. Organized / Tracked in Trello or Github (this depends on the nature of the project).

How this would work would be like this I am going to try and make a very universal idea of what this would look like.

------------

Idea is brought up in Discord. Team discusses it and the team rep or lead decides that the team wants to commit to that project.

Said project details are recorded in a shared Google Doc which is then posted into a Trello board for status tracking.

The team collectively works from that Google Doc and all assets, ideas, and discussion happen in that Document / Trello Card. 

Once the project is done and finalized it can be filed as done in Trello or in the Github and would be sent off.

--------------

For events or things that are private in nature (example events that are not public yet or partnership discussions that the other party does not want public) those can have a designated private board / tool.

The TLDR is basically we need to standardize the process of organizational tools across all teams to have one consistent process.

**Reply by Missy305** [reply]
Not gonna lie, I've never used trello nor github until Glimesh went there. Main information all in 1 place is important, but it doesn't have to be the same place private organizing v public information, and I think that cross occurred. I think Github brings to the table what drive and trello does with formatting, picture inserting, and thread organization, but I don't know all the ins and outs of trello for how that's being utilized.

**Comment by Heiwa1580** [comment]
Posting this for Kaz as he doesn't have a Github account and wasn't able to make one at the moment but wanted to get his thoughts out

Departmental organization can be difficult sometimes but because we are volunteers and not employees we should make it as simple as possible for anyone to join or leave due the nature of the work we do.

In terms of leadership. Because we are volunteers there will always be an influx of people coming and going from the team. This would make a vote almost impossible to have as there would not be a way to truly set the number of a "majority". If we were employees then this may be more feasible. This means that we more than likely need a leader. How do we choose the leader?
-It cannot be done with a popular vote as previously stated above. Someone could be missed or not heard
-there are a couple of options in my head. Paco and Clone could select leadership for each sector or people could submit to be the leader and Paco/Clone would decide.
-the second seems more feasible as anyone who volunteers for leadership will know that it will take their time. Clone/Paco choosing makes sense as they're the only 2 "employees" of the company.
-in the off chance that nobody volunteers, Paco/clone would have to take charge of these departments as the only employees working on this full time.
To me this makes the most sense. Ironically we may need a group consensus to this idea.

That being the case, I think the leadership is there to primarily handle disputes and make the final decision. They should not be in charge of setting up the foundation of the department. At first I was apprehensive on this idea that I came up with but let me explain. Whether it's a volunteer chosen or Paco/Clone, we don't know what will happen and if they'll be removed/leave/etc. This means that the way a department does things can change massively based on who's in charge. We need to have a uniform platform that all the departments work on even if they are in separate areas within that platform. Of course these can be changed with a wide discussion but it does not fall in the whim of who's in charge at the time. 

So in this spirit I have come up with some ideas myself.

The platform we should use: Trello. 
While I'm not a huge fan of trello because it's not easily accessible to me from mobile I think overall it will be the best platform to use in terms of keeping up with the projects and discussion. We mentioned Google docs, trello, Google sheets, discord. Google docs and sheets can easily be integrated into trello by separating them into columns for stages of the projects and using comments for updates and feedback. 

That leaves discord. There's plenty of chat in discord and we shouldn't let it go to just trello. If would inundate the platform and it's not as fluid.. So I also want to give some ideas on that. We want the discord to be hyper focused. I feel like the whole monitoring of what is in what chat can get a bit aggressive sometimes without finding out what the original chat was for. So here are my suggestions currently. 

-whenever someone chats in in those discord channels first they must add a topic line. State what you're referring to. Ex: map video, glimtip, other. 

That way people know that you are talking about official business. When correcting someone that what they posted may be in the wrong chat, the person who corrected should let the original poster explain why they believe it is in the right chat in case of the confusion. Although the topic line should help alleviate this issue. Anyone responding to this is up for debate whether they should also have the topic on their first sentence or not. I'm leaning against and keeping it only to who initiates the topic. 

In terms of keeping up with the discord, it can be challenging since there's so many conversations. I also propose potentially once a week having everyone say how their project is doing if they are doing something with the #update tag so it's easily searchable. Then everyone knows how the project is going every week. 

These are just some thoughts that I could put together on my phone. I hope they sound coherent. I may have missed some points as well so I may need to revise and update in a later post. 

Thanks, 
Kaz"
